### DISCUSSION TITLE
Cannot startup JanusGraph when using remote elasticsearch

### DISCUSSION THREAD

**Discussion by ericsodt** [JanusGraph/janusgraph#2993] [Q&A]
We previously had JG up and running without any issues using HBASE as our backend storage.  To try and speed up queries we stood up an elasticsearch cluster, however now our JG instance is not starting up, saying that "could not get type for name org.janusgraph.example.GraphApp".

![image](https://user-images.githubusercontent.com/13079554/159763603-8a5b6fdd-0fc3-4015-a052-5774fddd6f64.png)

Our elastic properties are setup like:
```
index.search.backend="elasticsearch"
index.search.hostname = "myhost.elasticsearch:9200"  ## https://myhost.elasticsearch:9200/janusgraph  <-- returns index values
index.search.elasticsearch.client-only=true
```
I've been banging my head on this one for a while.  Is there anything that I am missing??

**Comment by vtslab** [comment]
JanusGraph does not handle quotes in the properties files, so please try without.

**Reply by ericsodt** [reply]
Thanks, I tried that but got the same outcome

**Comment by vtslab** [comment]
Did you take into account the [mutability levels ](https://docs.janusgraph.org/configs/configuration-reference/#mutability-levels)of the graph?
Also beware of [lurking failed instances](https://docs.janusgraph.org/operations/recovery/#janusgraph-instance-failure) of the graph.

**Reply by ericsodt** [reply]
With the Gremlin console open, I am creating a new instance with a new hbase table, which previously did not exist.  It's during the call to .open() that I am getting these errors.  I'm not sure if mutability is an issue in this case as it should not have any instances running.

**Comment by vtslab** [comment]
Indeed, for a new graph with the HBase table nonexistent, mutability levels are not relevant. So, two more questions:
1. can you provide the entire properties file that you use?
2. can you still open a new graph with just HBase and no configs for elasticsearch?

Also, current JanusGraph versions do not seem to use the index.search.elasticsearch.client-only property anymore and do not have explicit default 9200 port number in example configs, but I do not expect that to be an issue.
